Changelog.md

```diff
@@ -46,6 +46,7 @@ Breaking Changes:
  * Type Checker: Disallow specifying base constructor arguments multiple times in the same inheritance hierarchy. This was already the case in the experimental 0.5.0 mode.
  * Type Checker: Disallow calling constructor with wrong argument count. This was already the case in the experimental 0.5.0 mode.
  * Type Checker: Disallow uninitialized storage variables. This was already the case in the experimental 0.5.0 mode.
+ * Type Checker: Detecting cyclic dependencies in variables and structs is limited in recursion to 256.
```

Comment:
Does that mean that otherwise the code is valid? I would rather put that into `bugfixes` - it is not really part of a language change, since we always have physical limitations of the compiler.

Reply:
It is indeed no language change but a preventing mechanism against crashing when the input is rather malicious formed. Not sure whether I'd put it under bugfixes but rather a behavioral/security-related change. I can do that in a mini-PR, but I feel like you'd be probably faster as you can review that one-liner yourself. yay?/nay? :-)